reproducibility_tests/README.md

```diff
@@ -18,15 +18,22 @@ Our solution to dealing with failure modes is by providing users with two workfl
    - [Update mse tables](#How-to-update-mse-tables)
 
  - A comparable reference dataset does **not** exists:
-   - Increment the reference counter in `reproducibility_tests/ref_counter.jl`. This triggers a "self-reference".
+   - Increment the reference counter in `reproducibility_tests/ref_counter.jl`.
    - [Update mse tables](#How-to-update-mse-tables) _all to zero values_
 
 At this moment, it's crucial to mention several important points:
 
- - When a reference dataset does not exist, we still perform a reproducibility test so that we continuously exercise the testing infrastructure. However, we compare the solution dataset with itself (which we call a "self-reference"). Therefore, _all reproducibility tests for all jobs will pass_ (no matter what the results look like) when the reference counter is incremented. So, it is important to review the quality of the results when the reference counter is incremented.
+ - When a reference dataset does not exist, we still perform a reproducibility test so that we continuously exercise the testing infrastructure. However, we compare the solution dataset with itself. Therefore, _all reproducibility tests for all jobs will pass_ (no matter what the results look like) when the reference counter is incremented. So, it is important to review the quality of the results when the reference counter is incremented.
 
  - Every time the reference counter is incremented, data from that PR is saved onto Caltech's central cluster. And that solution's dataset is the new reference dataset that all future PRs are compared against (until the reference counter is incremented again).
```

Comment:
Is this still accurate? (Are we saving data at each PR?)

Reply:
Ah, no, I missed that one in some recent commits. Updating...

Reply:
Wait, that looks stale

Reply:
Oh, I was looking at the wrong section. Yes, we move data every time the reference is incremented.

Reply:
N references. We only compare against merged commits (i.e., on the main branch)

Reply:
> N references. We only compare against merged results (i.e., on the main branch): only the head commit is moved to central, so only 1 folder should be moved per PR.

I am still confused.

N references, as the previous N times the ref counter was incremented?
Or the previous N PRs that were merged (ie, the latest N commits on main)?

Reply:
Yeah, sorry, I know it's a bit confusing. It's N references, not N times the ref counter was incremented, that's just an ever increasing number (and we garbage collect old results after a while). Maybe an example will help:

```
187 # PRs against ref counter 187 has 3 references
├── 73hasd
├── fgsae7
└── sdf63a
188 # PRs against ref counter 188 has 2 references
├── 73hasd
└── sdf63a
189 # PRs against ref counter 189 has 4 references
├── 73hasd
├── sd9g8y
├── uhw45l
└── sdf63a
```

Reply:
Okay, perfect! This is very helpful and I think we should have in the README.

Reply:
Ok, I'll add a bit about this, in anticipation of changing our folder structure.